### DISCUSSION TITLE
Meteorological Data

### DISCUSSION THREAD

**Discussion by fidaulmustafa** [phydrus/phydrus#48] [Q&A] [answered]
Hi guys, I needed to ask if Phydrus has been updated to account for the meteorological data such as wind, air temps etc?

I think the recent version that I downloaded, does not take in the meteorological information. Can you please guide me regarding that?
I am looking to use Phydrus for groundwater recharge estimate and was hoping to find a way to add the meteorological information into my study. 

Thanks.

**Comment by martinvonk** [comment, marked as answer]
What kind of boundary condition would you like to use? If you are just using a atmospheric boundary condition, you only need rSoil (soil evaporation), rRoot (transpiration) and Prec (precipitation). You can calculate those from the potential evapotranpiration which can in turn be calculated from your meteorological data using [phydrus/pyet](https://github.com/phydrus/pyet)

**Reply by LIZAO-coder** [reply]
Thank you very much for your answer.
ml.add_waterflow(model=0, linitw=True, top_bc=5, bot_bc=4, ha=1e-6, hb=1e4)
If the upper boundary condition is set to Variable Pressure Head/Flux or others, i.e., top_bc=5, how should the relevant parameters be input?Is it also using the Model.add_atmospheric_bc function? How should it be set?
